__tests__/app.test.js

```diff
@@ -70,6 +70,7 @@ describe("GET /api/articles/:article_id", () => {
           votes: 100,
           article_img_url:
             "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+          comment_count: '11'
```

Comment:
I would add a new test rather than changing a previous one :)  this is why its useful to use flexible matchers like toMatchObject so if properties are added they dont break previous tests !